src/externalized/ItemSystem.h

```diff
@@ -8,4 +8,5 @@ class ItemSystem
 {
 public:
 	virtual const ItemModel* getItemByName(const ST::string &internalName) const = 0;
+	virtual const ItemModel* getItem(uint16_t itemIndex) const = 0;
```

Comment:
This looks redundant. Did you mean to add a non-static GetItemFromValue instead?

Reply:
This is to expose an existing method in DefaultContentManager via ItemSystem interface. 

It probably would touch many more files if we rename it

Reply:
This meant to expose an existing method in `DefaultContentManager` via the `ItemSystem` interface.

https://github.com/ja2-stracciatella/ja2-stracciatella/blob/2ae26c4e46a1c268bd93af7e8d5573090b2ae797/src/externalized/DefaultContentManager.cc#L1005

It would probably touch many more files if we are to rename it.

Reply:
Ah, missed the one use later down, never mind.